test/integration/remote_installer_int_test.go

```diff
@@ -74,7 +74,7 @@ func (suite *RemoteInstallIntegrationTestSuite) TestInstall() {
 			installPath := filepath.Join(ts.Dirs.Work, "install")
 			stateExePath := filepath.Join(installPath, "bin", constants.StateCmd+osutils.ExeExtension)
 
-			args := []string{"-n"}
+			args := []string{"--non-interactive"}
```

Comment:
Did `-n` not work?

Reply:
It did, but I like that this is more explicit.